actions/minetest

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/python3
+# -*- mode: python -*-
+#
+# This file is part of Plinth.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""
+Configuration helper for Minetest server.
+"""
+
+import argparse
+import augeas
+
+from plinth import action_utils
+
+
+CONFIG_FILE = '/etc/minetest/minetest.conf'
+AUG_PATH = '/files' + CONFIG_FILE + '/.anon'
+
+def load_augeas():
+    """Initialize Augeas."""
+    aug = augeas.Augeas(flags=augeas.Augeas.NO_LOAD +
+                        augeas.Augeas.NO_MODL_AUTOLOAD)
+    aug.set('/augeas/load/Php/lens', 'Php.lns')
+    aug.set('/augeas/load/Php/incl[last() + 1]', CONFIG_FILE)
+    aug.load()
+    return aug
+
+aug = load_augeas()
+
+def parse_arguments():
+    """Return parsed command line arguments as dictionary"""
+    parser = argparse.ArgumentParser()
+    subparsers = parser.add_subparsers(dest='subcommand', help='Sub command')
+
+    configure = subparsers.add_parser('configure', help='Configure Minetest')
+    configure.add_argument('--max_players', help='Set maximum number of players')
+    configure.add_argument('--creative_mode', choices=['true', 'false'],
+                           help='Set creative mode true/false')
+    configure.add_argument('--enable_pvp', choices=['true', 'false'],
+                           help='Set player Vs player true/false')
+    configure.add_argument('--enable_damage', choices=['true', 'false'],
+                           help='Set damage true/false')
+
+    return parser.parse_args()
+
+def subcommand_configure(arguments):
+    """Configure Minetest."""
+    if arguments.max_players is not False:
```

Comment:
We need to make a change here.

```sudo ./actions/minetest configure --creative_mode false``` should not cause the max_users setting in the config file to change. But it does, currently, it clears the value that was set.

The problem is not on the Django side: it is calling the script correctly.